docs/source/getstarted.rst

```diff
@@ -27,12 +27,13 @@ Please visit our `Cozmo SDK Forums <https://forums.anki.com/>`_ where you can:
 Starting Up the SDK
 -------------------
 
-1. Plug the mobile device containing the Cozmo app into your computer.
-2. Open the Cozmo app on the phone. Make sure Cozmo is on and connected to the app via WiFi.
-3. Tap on the gear icon at the top right corner to open the Settings menu.
-4. Swipe left to show the Cozmo SDK option and tap the Enable SDK button.
-5. On the computer, open a Terminal (macOS/Linux) / Command Prompt (Windows) window. Type ``cd cozmo_sdk_examples`` where *cozmo_sdk_examples* is the name of the directory you extracted the SDK examples into and press Enter. The SDK example scripts can be downloaded from the :doc:`Downloads page </downloads>`.
-6. Now you can run any of the example programs.
+1.	Plug the mobile device containing the Cozmo app into your computer.
+2.	Open the Cozmo app on the phone. Make sure Cozmo is on and connected to the app via WiFi.
+3.	Tap on the gear icon at the top right corner to open the Settings menu.
+4.	Swipe left to show the Cozmo SDK option and tap the Enable SDK button.
+5.	Make sure the SDK examples are downloaded from the :doc:`Downloads page </downloads>`.  Remove the version number from the folder name so that it is just called “cozmo_sdk_examples”.
```

Comment:
Added the bit about removing the version number in case people tried to navigate to the folder "cozmo_sdk_examples" when the folder is actually called "cozmo_sdk_examples_0.14.0".  But if it hasn't been a source of confusion then I guess I was worried for nothing :)